### ISSUE TITLE
Magic lamp animation broken

### ISSUE THREAD

**Issue by sloiy** [psifidotos/Latte-Dock#1028]
Hello,
I have noticed recently that when I open Thunderbird (and Firefox sometimes), the minimize animation with magic lamp enabled is kind of broken.
And with the default minimization animation, it has a bug too : it chooses another animation for minimizing.
I have only seen this bug with Latte-Dock. With a normal task manager, the magic lamp animation is working correctly.

Here is the video of the broken magic lamp animation with Thunderbird :
[animation_latte.zip](https://github.com/psifidotos/Latte-Dock/files/2020751/animation_latte.zip)

Thanks in advance :)

**Comment by sloiy** [comment]
Hello,
1. It is happening with Thunderbird mainly, and sometimes with Firefox. Only these two packages, to my knowledge.
2. I'm using v0.7 provided by Gentoo repository.

So do you think this could be a Plasma-related bug ?

**Comment by psifidotos** [comment]
yes, but someone must prove it first and you should test also with master version

**Comment by sloiy** [comment]
I have tried with master version and I think it is working now. No more bugs with magic lamp and Thunderbird/Firefox.
Thank you
